x/fswap/keeper/keeper.go

```diff
@@ -15,18 +15,23 @@ import (
 )
 
 type Keeper struct {
-	cdc       codec.BinaryCodec
-	storeKey  storetypes.StoreKey
-	config    types.Config
-	authority string
+	cdc        codec.BinaryCodec
+	storeKey   storetypes.StoreKey
+	config     types.Config
+	authority  string
+	authkeeper AccountKeeper
 	BankKeeper
 }
 
-func NewKeeper(cdc codec.BinaryCodec, storeKey storetypes.StoreKey, config types.Config, authority string, bk BankKeeper) Keeper {
+func NewKeeper(cdc codec.BinaryCodec, storeKey storetypes.StoreKey, config types.Config, authority string, ak AccountKeeper, bk BankKeeper) Keeper {
```

Comment:
Why does the `AccountKeeper` need? I think it is not used in keeper. If `AccountKeeper` only needs for testing, please move it to test code.

Reply:
The `AccountKeeper` is being used only inside of `NewKeeper()`. So it won't be necessary to add it into the `Keeper` structure. The signature itself, on the other hand, has no problem in my opinion.